package.json

```diff
@@ -43,6 +43,7 @@
     "@types/google-protobuf": "^3.15.5",
     "@types/node-fetch": "^2.6.2",
     "body-parser": "^1.19.0",
+    "cloudevents": "^6.0.3",
```

Comment:
Do we need to add this dependency? I propose that we stick to "required" fields on CloudEvents to check for it, and return a simple object, letting users cast it to CloudEvent objects if required.

Reply:
Removed it :)

Reply:
Thank you!